vms/platformvm/state/state.go

```diff
@@ -292,9 +305,17 @@ type state struct {
 	currentSupply, persistedCurrentSupply uint64
 	// [lastAccepted] is the most recently accepted block.
 	lastAccepted, persistedLastAccepted ids.ID
+	indexedHeights                      *heightRange
 	singletonDB                         database.Database
 }
 
+// heightRange is used to track which heights are safe to use the native DB
```

Comment:
Because we aren't re-indexing old heights, I'm not exactly sure when we would remove this... We would need to wait to remove this until all subnets P-chain heights have progressed past the time that all nodes have updated to the new DB index.

Reply:
might be worth elaborating on this in some sort of TODO?

Reply:
added